source/backups.rst

```diff
@@ -138,6 +138,41 @@ When the backup destination is configured and applied with `kubectl apply -f dep
         storageName: s3-us-west
       EOF
 
+.. _backups-pitr-binlog:
+
+Backing up binary logs for point-in-time recovery
```

Comment:
Please mention that this is supported only with PXC version `8.0.21-12.1` and higher.